debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9ObjectHelper.java

```diff
@@ -44,7 +48,8 @@
 	private static int[] counts;
 	private static long probes;
 	private static long hits;
-	
+	public static boolean mixedModeSupport = AlgorithmVersion.getVersionOf(AlgorithmVersion.GC_OBJECT_HEADER_VERSION).getAlgorithmVersion() > 0;
```

Comment:
Why not use `OMR_GC_COMPRESSED_POINTERS` and `OMR_GC_FULL_POINTERS` instead of adding the otherwise unused `GC_OBJECT_HEADER_VERSION`?

Reply:
FULL pointers is not a real build flag yet.

Reply:
Also it doesn't exist in older cores at all.

Reply:
`OMR_GC_FULL_POINTERS` is all over the place in OMR; this is just the first use I found: [1]. We have `J9ConfigFlags` that can help with its absence in older core files.

[1] https://github.com/fjeremic/omr-clang-format-automation/blob/8ff5b9502099b609b5644ac2484cd9b80bdf5f98/gc/base/MemoryPool.hpp#L67

Reply:
It's not a build flag as such - it's something I define in omr.h for now. I'm not changing this in this PR (particularly since it involves an OMR change).

Reply:
It doesn't need to be a 'build flag'; all we care is that it is a constant defined in the blob (or not).